src/Action/NotifyAction.php

```diff
@@ -22,18 +28,20 @@ final class NotifyAction implements ActionInterface, ApiAwareInterface
     private Request $request;
 
     public function __construct(
-        private readonly RequestStack $requestStack,
+        protected RequestStack $requestStack,
```

Comment:
I think the issue there was 'readonly' keyword, not the 'private'. Could you check whether the private, but not readonly would work correctly?